src/isa/instr.rs

```diff
@@ -644,18 +644,15 @@ pub enum BytesOp {
     ///
     /// If the string register is empty, or destination register can't fit the length, sets `st0`
     /// to `false` and destination register to `None`.
-    #[display("len     {0},{1}{2}")]
+    #[display("len     {1}{2},{0}")]
     Len(/** `s` register index */ RegS, RegA, Reg32),
 
     /// Count number of byte occurrences from the `a8` register within the string and stores that
     /// value into destination `a16` register.
     ///
-    /// If the string register is empty, or destination register can't fit the length, sets `st0`
-    /// to `false` and destination register to `None`.
-    ///
-    /// If the source byte value register is uninitialized, sets destination register to `None` and
-    /// `st0` to `false`.
-    #[display("cnt     {0},a8{1},a16{2}")]
+    /// If the string register is empty, or the source byte value register is uninitialized, sets
+    /// `st0` to `false` and destination register to `None`.
+    #[display("cnt     a16{2},{0},a8{1}")]
     Cnt(
```

Comment:
unlike `len`, the return value cannot overflow the destination register which is always a16 in `cnt`